### PR TITLE
Eobs 753 high clinical risk process change

### PR DESCRIPTION
**Pull request by Gimpneek**
- Making test_util objects a little more unique

### CODE REVIEW

File: nh_clinical/tests/common/nh_clinical_test_utils.py [hunk 1]
```diff
@@ -166,3 +198,32 @@ def copy_instance_variable_if_exists(self, caller, variable_name):
         instance_variable_value = getattr(self, variable_name, None)
         if instance_variable_value:
             setattr(caller, variable_name, instance_variable_value)
+
+    # TODO EOBS-753: Move to patient model? Is just a generally useful method.
```
Comment (moderatemisbehaviour) [inline nh_clinical/tests/common/nh_clinical_test_utils.py]: This method isn't really test specific and may be useful in general, would it be worth moving to the patient model?
You could always keep this method and just call to the implementation in the patient model for convenience.

Comment (moderatemisbehaviour) [inline nh_clinical/tests/common/nh_clinical_test_utils.py]: Sorry put the comment in a bad place, I'm referring to the `get_open_activities_for_patient` method.

Comment (Gimpneek) [inline nh_clinical/tests/common/nh_clinical_test_utils.py]: TBH this method's kind of misnamed as it's not looking at the `patient_id` but the `parent_id` (which is the spell's activity ID).

Additionally there's a few defaults in there such as `self.nurse.id` that make it a little hard to move into the patient model

**Comment by moderatemisbehaviour** [comment]
LGTM :shipit:

**Comment by Nayira** [comment]
Test for EOBS-753 passed :shipit:

**Pull request merged**